RecoMET/METAlgorithms/src/GlobalHaloAlgo.cc

```diff
@@ -48,13 +48,18 @@ reco::GlobalHaloData GlobalHaloAlgo::Calculate(const CaloGeometry& TheCaloGeomet
   float METOverSumEt = TheCaloMET.sumEt() ? TheCaloMET.pt() / TheCaloMET.sumEt() : 0 ;
   TheGlobalHaloData.SetMETOverSumEt(METOverSumEt);
 
-  //int EcalOverlapping_CSCRecHits[73];
-  //int EcalOverlapping_CSCSegments[73];
-
   int EcalOverlapping_CSCRecHits[361];
```

Comment:
Why not `int EcalOverlapping_CSCRecHits[361] = {};`? I think I have done something like that before, and it zero-initializes all elements.

Reply:
Why not?
That's cleaner, thank you @guitargeek